ux.symfony.com/config/packages/web_profiler.yaml

```diff
@@ -1,6 +1,6 @@
 when@dev:
     web_profiler:
-        toolbar: true
+        toolbar: false
```

Comment:
Not needed

Reply:
Was needed in dev while https / firefox was buggy :) 

Reverted